readme.md

```diff
@@ -1,20 +1,109 @@
 <p align="center">
-  <img src="https://docs.ai-jsx.com/img/ai-jsx.png" alt="AI.JSX logo" width="300" />
+  <img src="https://docs.ai-jsx.com/img/ai-jsx.png" alt="AI.JSX logo" width="100" />
 </p>
 
-# AI.JSX
+# AI.JSX — The AI Application Framework for Javascript
 
-## Getting Started
+[![Docs Site](https://img.shields.io/badge/Docs%20Site-docs.ai--jsx.com-orange)](https://docs.ai-jsx.com)
+[![Discord Follow](https://dcbadge.vercel.app/api/server/MsKAeKF8kU?style=flat)](https://discord.gg/MsKAeKF8kU)
+[![Twitter Follow](https://img.shields.io/twitter/follow/fixieai?style=social)](https://twitter.com/fixieai)
```

Comment:
https://youtu.be/cP-rQb_daYE?t=63

Reply:
There are more of these, if you care to add them, for weekly npm installs, listing the latest version on npm, etc.